spec/app/api_spec.rb

```diff
@@ -22,43 +23,588 @@ def validate!(schema, json)
   # JSON schema for collections where we only send back object references;
   # these are the same no matter what the underlying collection elements
   # look like
-  ObjectRefCollectionSchema = {
-    '$schema'  => 'http://json-schema.org/draft-04/schema#',
-    'title'    => "Object Reference Collection JSON Schema",
+  ObjectRefItemSchema = {
     'type'     => 'object',
     'additionalProperties' => false,
     'properties' => {
       "spec" => {
         'type'    => 'string',
         'pattern' => '^https?://'
       },
-      "items" => {
```

Comment:
Is `items` going away here? I thought this wouldn't be an API change at all, just an addition.

Reply:
No, that is the schema for an item in a collection that's an object ref. See the collection_schema routine later below api_spec.

Otherwise if you look at collection_view in lib/razor/view.rb, the API's unchanged.